samples/CommunityToolkit.Maui.Sample/Pages/Views/SemanticOrderView/SemanticOrderViewPage.xaml.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+using CommunityToolkit.Maui.Sample.ViewModels.Views;
+namespace CommunityToolkit.Maui.Sample.Pages.Views;
+
+public partial class SemanticOrderViewPage : BasePage<SemanticOrderViewPageViewModel>
+{
+	public SemanticOrderViewPage(SemanticOrderViewPageViewModel vm) : base(vm)
+	{
+		InitializeComponent();
+		acv.ViewOrder = new List<View> { first, second, third, fourth, fifth };
```

Comment:
What is the recommended way for users to implement `ViewOrder`? 

If it's with a ViewModel bindings, let's move this to `SemanticOrderPageViewModel`.
If it's in XAML, let's move this to XAML.